impeller/compiler/switches.h

```diff
@@ -36,6 +36,7 @@ struct Switches {
   uint32_t gles_language_version;
   std::string metal_version;
   std::string entry_point;
+  bool use_half_textures;
```

Comment:
Default initialize all members. This and (if you are in the mood for drive-by changes) the other members in the struct.

Reply:
Done